main.c

```diff
@@ -3,9 +3,93 @@
 #include <string.h>
 #include "mzcc.h"
 
+static char *outfile = NULL, *infile = NULL;
+extern FILE *outfp;
+static bool dump_ast;
+
+static void usage()
+{
+    fprintf(stdout,
+            "mzcc [options] filename\n"
+            "OPTIONS\n"
+            "  -o filename            Write output to the specified file.\n"
+            "  --dump-ast             Dump abstract syntax tree(AST)\n");
+}
+
+void print_usage_and_exit()
```

Comment:
At present, function `print_usage_and_exit` is not exposed. So, mark it as `static` as well.